package.json

```diff
@@ -309,6 +309,11 @@
           "when": "editorLangId == sql && resourcePath in mssql.runningQueries",
           "group": "navigation@2"
         },
+        {
+          "command": "mssql.revealQueryResultPanel",
+          "when": "editorLangId == sql && view.queryResult.visible == false",
```

Comment:
does this `when` clause also need to check `config.mssql.enableRichExperiences`?

Reply:
hah, you fixed it already

Reply:
I've actually updated this clause, you might need to refresh the page.